Gemfile.lock

```diff
@@ -44,61 +44,60 @@ GEM
       jsonapi-renderer (>= 0.1.1.beta1, < 0.3)
     active_storage_base64 (1.0.0)
       rails (~> 6.0)
-    activeadmin (2.3.1)
+    activeadmin (2.6.0)
       arbre (~> 1.2, >= 1.2.1)
       formtastic (~> 3.1)
       formtastic_i18n (~> 0.4)
       inherited_resources (~> 1.7)
       jquery-rails (~> 4.2)
       kaminari (~> 1.0, >= 1.0.1)
-      railties (>= 5.0, < 6.1)
+      railties (>= 5.2, < 6.1)
       ransack (~> 2.1, >= 2.1.1)
       sassc-rails (~> 2.1)
       sprockets (>= 3.0, < 4.1)
-      sprockets-es6 (~> 0.9, >= 0.9.2)
-    activejob (6.0.0)
-      activesupport (= 6.0.0)
+    activejob (6.0.2.1)
+      activesupport (= 6.0.2.1)
       globalid (>= 0.3.6)
-    activemodel (6.0.0)
-      activesupport (= 6.0.0)
-    activerecord (6.0.0)
-      activemodel (= 6.0.0)
-      activesupport (= 6.0.0)
-    activestorage (6.0.0)
-      actionpack (= 6.0.0)
-      activejob (= 6.0.0)
-      activerecord (= 6.0.0)
+    activemodel (6.0.2.1)
+      activesupport (= 6.0.2.1)
+    activerecord (6.0.2.1)
+      activemodel (= 6.0.2.1)
+      activesupport (= 6.0.2.1)
+    activestorage (6.0.2.1)
+      actionpack (= 6.0.2.1)
+      activejob (= 6.0.2.1)
+      activerecord (= 6.0.2.1)
       marcel (~> 0.3.1)
-    activesupport (6.0.0)
+    activesupport (6.0.2.1)
       concurrent-ruby (~> 1.0, >= 1.0.2)
       i18n (>= 0.7, < 2)
       minitest (~> 5.1)
       tzinfo (~> 1.1)
-      zeitwerk (~> 2.1, >= 2.1.8)
+      zeitwerk (~> 2.2)
     addressable (2.7.0)
       public_suffix (>= 2.0.2, < 5.0)
-    airbrussh (1.3.4)
+    airbrussh (1.4.0)
       sshkit (>= 1.6.1, != 1.7.0)
-    annotate (3.0.2)
+    annotate (3.0.3)
```

Comment:
### [annotate](http://github.com/ctran/annotate_models)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 3.0.2 → 3.0.3

[_~~change-log~~, [source-code](https://github.com/ctran/annotate_models)_]





<details>
<summary>Commits</summary>

A change of **27** commits. See the full changes on [the compare page](https://github.com/ctran/annotate_models/compare/v3.0.2...v3.0.3).

These are the first 10 commits:
- (d5d55e8) [Merge branch 'hotfix/v2.6.7'](https://github.com/ctran/annotate_models/commit/d5d55e8bcb4459c66c7ae50c911ab1189ab585d5)
- (52ffefc) [Merge branch 'release/v2.6.8'](https://github.com/ctran/annotate_models/commit/52ffefcbe4e89cb172e2007eda0f74ac40798727)
- (1de5bfb) [Merge branch 'release/v2.6.9'](https://github.com/ctran/annotate_models/commit/1de5bfbc0e4df8f66da8a85435f7eb8c15e811ab)
- (0146ee7) [Merge branch 'release/v2.6.10'](https://github.com/ctran/annotate_models/commit/0146ee7b9a70d530a30a5e220c94d5e353624671)
- (7b33065) [Runs annotate on 'rake db:rollback' too](https://github.com/ctran/annotate_models/commit/7b330655899417c144a58b0c86958cb4a33ae168)
- (1ee47e5) [Merge pull request #249 from agoodno/run-on-rollback](https://github.com/ctran/annotate_models/commit/1ee47e53eb69cd05f41aa921dcec2b197bea7a16)
- (8f3aeb1) [Revert "Runs annotate on 'rake db:rollback' too"](https://github.com/ctran/annotate_models/commit/8f3aeb1002d168218071087af98cd5fa96904d63)
- (d878a5b) [Merge pull request #259 from ctran/revert-249-run-on-rollbac…](https://github.com/ctran/annotate_models/commit/d878a5b784ee0f703e9ee677bc0231dc24159512)
- (0b05456) [Merge branch 'release/v2.7.0'](https://github.com/ctran/annotate_models/commit/0b054562b8e5564d8de04f3398053e06daa6d426)
- (5f37a97) [Merge branch 'release/v2.7.2'](https://github.com/ctran/annotate_models/commit/5f37a974f3dff68f92b3495115f1399e7db5c584)


</details>